workshop/tools/pods

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
what is the goal of this script?

Reply:
:-)  sorry meant to add that.  It's just a nicer way to show the pods & ksvcs running during a live demo of Knative

Reply:
fixed - if you don't see value in this I can drop it, but I do use it myself so I thought I'd share